build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+	java
+	id("org.springframework.boot") version "3.3.4"
+	id("io.spring.dependency-management") version "1.1.6"
+}
+
+group = "com.softdevsix"
```

Comment:
What about using "com.argos" instead. SoftDevSix is the name of our organization

Reply:
the structure is as follows com.softdevsix.argos, I don't know if you want me to change the structure? and have it be like this com.argos.softdevsix? or just com.argos?

Reply:
![image](https://github.com/user-attachments/assets/f971a5ea-a634-4c0f-81c1-4b590590abc0)
After careful observation I realized the group name, which I think it's fine. So I'll mark this as resolved.